src/main/kotlin/no/nav/syfo/App.kt

```diff
@@ -70,5 +70,9 @@ fun main() {
         }
     )
 
+    server.environment.monitor.subscribe(ApplicationStarted) { application ->
```

Comment:
@janzh Tror dette er er bedre måte å sette ready på (dvs sånn at det ikke settes før db-modulen og app-modulen er initialisert)

Reply:
Ser ut som om resultatet blir det samme, men det ser jo litt ryddigere ut ja 👍

Reply:
@janzh Resultatet blir **ikke** det samme: Slik det var settes ready=true for tidlig, dvs før databasen og api'et er initialisert. Dette skjer fordi disse initialiseringene av modulene spinnes av i subrutiner.